modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -3079,7 +3177,7 @@ organizeSubjectData <- function(
                                             lockCorpBatchId= F, errorEnv= errorEnv, precise = precise, link = link, 
                                             calculateGroupingID = preciseTreatmentGroupID, 
                                             stateAssignments = stateAssignments, concColumn = concColumn,
-                                            codeAssignments = codeAssignments)
+                                            codeAssignments = codeAssignments)[[1]]
```

Comment:
might be safer to not assume it's the first one in this list but rather specific select the subject data, but up to you